corehq/apps/data_interfaces/models.py

```diff
@@ -73,36 +73,31 @@ def get_case_ids(cls, domain, case_type, boundary_date=None):
         Retrieves the case ids in chunks, yielding a list of case ids each time
         until there are none left.
         """
-        offset = 0
-        chunk_size = 10000
+        chunk_size = 100
 
-        # We have to sort on _uid; _id is not sortable. See:
-        # https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-uid-field.html
-        # https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-id-field.html
+        query = (CaseES()
+                 .domain(domain)
+                 .case_type(case_type)
+                 .is_closed(closed=False)
+                 .exclude_source()
+                 .size(chunk_size))
 
-        while True:
-            query = (CaseES()
-                     .domain(domain)
-                     .case_type(case_type)
-                     .is_closed(closed=False)
-                     .exclude_source()
-                     .sort('_uid')
-                     .start(offset)
-                     .size(chunk_size))
+        if boundary_date is not None:
+            query = query.server_modified_range(lte=boundary_date)
 
-            if boundary_date is not None:
-                query = query.server_modified_range(lte=boundary_date)
+        result = []
 
-            results = query.run()
-            ids = results.doc_ids
+        for case_id in query.scroll():
```

Comment:
Was this due to the ES issues we've had on prod recently? 100 seems pretty low.

The code here looks very similar, but if there's not a scale issue, you could use `get_ids` https://github.com/dimagi/commcare-hq/blob/3dbdc2dd6d9dab3ecdc29093b991945d71bd06d1/corehq/apps/es/es_query.py#L429

Alternatively we could change get_ids to use scroll if it's large.